utils/python/utils.go

```diff
@@ -0,0 +1,70 @@
+package python
+
```

Comment:
It looks nlike the code in this file can be merged with the code in ```utils/python/piputils.go ```. Not sure if there's a need for this split.
Let's also merge the ```populatePythonDependencyTree``` utility function into this single util file.